opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement.py

```diff
@@ -12,6 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from dataclasses import dataclass
+from typing import Optional, Union
 
+from opentelemetry.util.types import Attributes
+
+
+@dataclass(frozen=True)
 class Measurement:
-    pass
+    value: Union[int, float]
+    attributes: Optional[Attributes] = None
+
+
+# Can't make Exemplar a subclass of Measurement because that makes
+# time_unix_nano to be defined after attribute, a keyword argument
+@dataclass(frozen=True)
+class Exemplar:
+    value: Union[int, float]
+    time_unix_nano: int
+    span_id: int
+    trace_id: int
+    attributes: Optional[Attributes] = None
+    filtered_attributes: Optional[Attributes] = None
```

Comment:
Can we remove the exemplars stuff for a separate PR?

Reply:
Sure, moving them out :+1: